README.md

```diff
@@ -57,113 +69,158 @@ Subdirectory README files should follow the same general format as this root fil
 
-No Authorization (Public method)
+The "address_type"
 
-Gets a new JWT for the given user. The JWT can be used to authenticate with the other endpoints. JWT should last 5 minutes.
+<hr>
+
+## TIER 2 ENDPOINTS
+
+The following endpoints are required for a server to be CAP 2 compliant. These endpoints allow a wallet to know how to interact with a server on behalf of the user, in other words CREATE/UPDATE/DELETE address/user info.
+
+### POST /v1/address
+
+Authorization: Bearer {jwt}
+
+Adds or updates the address for the authenticated user of the given coin type.
 
 | Parameters | Description | Required | Sample Value |
 | ---------- | ----------- | -------- | ------------ |
-| username | The username of the user | Yes | "lane"
-| password | The password of the user | Yes | "SU93Rc00lpa55"
+| ledger_id | The ledger_id for this address | Yes | 1 (BTC_SEGWIT)
+| address | The user's address for this ledger | Yes | bc1qvw0ytfntx6zs0lfsruem6xwj0mewng523ktatp
 
-#### Example Response
+#### Example Usage
+
+* Alice recieves a payment from Bob and her wallet recognizes that a payment was recieved
+* Assuming Alice wants to update her address, her wallet will generate a new address
+* Alice's wallet uses her alias to construct a URL and HTTP Method
+* alice@ogdolo.com -> POST <https://opencap.ogdolo.com/address>
+* Alice's wallet authenticates with the CAP server using her credentials to get a JWT that will securely allow her wallet to update her address
+* Alice's wallet constructs the JSON request to send to the server
 
 ```javascript
-Code: 200
+Authorization: Bearer {jwt}
 {
-    "jwt": "fn9s8dfasp7fub2uifb928rbpfdsfgasd9f8sdfbsa9fbsdf8ushdf9wbf"
+    "ledger_id": 0,
+    "address": "bc1qvw0ytfntx6zs0lfsruem6xwj0mewng523ktatp",
 }
 ```
 
-## Optional Endpoints
+* Alice sends the request and her address is updated on the server
 
-The following endpoints are not REQUIRED to adhere to CAP, because most of those functions could also be handled by a webpage portal, or simply on the server backend side if the user is running their own server. However, IF a server or wallet are going to implement the functionality of the following endpoints, they should do it in this manner. If for some reason this is impossible, please submit an issue/PR so we can fix the problem.
 
-### POST /user
+### POST /v1/auth
 
 No Authorization (Public method)
 
-Create a new user on the server.
+Gets a new JWT for the given user. The JWT can be used to authenticate with the other endpoints. JWT should last 5 minutes.
 
 | Parameters | Description | Required | Sample Value |
 | ---------- | ----------- | -------- | ------------ |
 | username | The username of the user | Yes | "lane"
 | password | The password of the user | Yes | "SU93Rc00lpa55"
-| email | The email of the user | Yes | "lane.wagner@gmail.com"

```

Comment:
We might want to keep the email field (optional) to allow resetting the password.

Reply:
I think it should be left out just because many servers won't use email. There is no reason to have two methods of auth.

Reply:
I did not mean this as auth method but as a way to reset the password, i.e. you click 'Forgot password', enter your username and get an email.  
But you're correct that mosts servers won't mail.

Reply:
yeah. leave it out.